pandas/core/generic.py

```diff
@@ -4275,7 +4275,29 @@ def get_ftype_counts(self):
 
     @property
     def dtypes(self):
-        """Return the dtypes in this object."""
+        """
+        Return the dtypes in this object.
+
+        Notes
+        -----
+        It returns a Series with the data type of each column.
```

Comment:
Maybe replace "it" with "This method". And let's say what the values and index is.

    This returns a Series with the data type of each column. The result's index is the
    original DataFrame's columns.

Let's also replace all instances of "object" with "DataFrame". I'm not sure why this is in generic.py since I think it's specific to DataFrame.

Reply:
I'm modifying dtype `@property` of the NDFrame in generic.py
Is "This property" okay instead of "This method"?

At first, ``object`` was ``NDFrame``, but found an error.

```
Private classes (['NDFrame']) should not be mentioned in public docstring.
```

but it seems to be better that the way you specify with "DataFrame", I will do that.